### PR TITLE
Include our limit for file upload in the API docs + 413 error code

### PR DESCRIPTION
**Pull request by Go-Fred**

### CODE REVIEW

**Comment by Go-Fred** [comment]
@jpjoyal @jugarrit can you review my PR when you get a time?

File: slate/source/includes/rest/core/_conversations.md [hunk 1]
```diff
@@ -1220,6 +1220,10 @@ Upload an attachment to Smooch to use in future messages. Files are uploaded usi
 |------------------------------|----------------------------|
 | **access**<br/><span class='req'>required</span>    | The access level for the attachment. Currently the only available access level is `public` |
 
+<aside class="notice">
+The maximal size allowed per file is 10 Mb. Exceeding this size will result in a [`413 error`](#errors).
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_conversations.md]: maximal -> maximum

Also I think it's 10MB not Mb

File: slate/source/includes/rest/core/_introduction.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ Smooch uses standard HTTP status codes to communicate errors
 | **403** | Forbidden - Your credentials are valid but you don't have access to the requested resource. |
 | **404** | Not Found - The object you're requesting doesn't exist. |
 | **409** | Conflict - You might be trying to update the same resource concurrently. |
+| **413** | Request entity too large - The size of the entity (e.g. file) you are trying to send exceeds our limit. |
```
Comment (mspensieri) [inline slate/source/includes/rest/core/_introduction.md]: I would change this to:

> Request entity too large - The request body exceeds the maximum allowed size for the current endpoint

This error doesn't only apply to files

**Comment by Go-Fred** [comment]
@mspensieri Done! Thanks!

**Pull request merged**